.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install
+        run: |
+          npm install
+      - name: Check licenses
+        run: |
+          npm run license-checker
+      - name: Run tests
+        run: |
+          npm run test:report
+      - name: Generate coverage
+        run: |
+          npm run coveralls -- -c
+      - name: Coveralls
```

Comment:
I'm literally working on doing this in the ldapjs repo right now. You need to use the parallel version -- https://github.com/coverallsapp/github-action#complete-parallel-job-example

Reply:
Only for performance or what's the issue?

Reply:
I got it due to matrix.